### PR TITLE
Add error notification for failed wallpaper downloads

### PR DESCRIPTION
**Pull request by Lucki**

### CODE REVIEW

File: src/notifications.ts [hunk 1]
```diff
@@ -16,6 +16,20 @@ class Notification {
         const message = `A new wallpaper was set!\n${infoString}`;
         notify('New Wallpaper', message);
     }
+
+    /**
+     * Show an error notification for failing to download the next wallpaper.
```
Comment (ifl0w) [inline src/notifications.ts:21]:
```suggestion
     * Show an error notification for failed wallpaper downloads.
```

File: src/notifications.ts [hunk 2]
```diff
@@ -16,6 +16,20 @@ class Notification {
...
+     *
+     * @param {unknown} error The error that was thrown when fetching a new wallpaper
+     */
+    static fetchWallpaperFailed(error: unknown): void {
+        let errorMessage = String(error);
+
+        if (error instanceof Error)
+            errorMessage = error.message;
+
+        notify('Download Failed!', errorMessage);
```
Comment (ifl0w) [inline src/notifications.ts:31]: We probably should add bit more context here in the title where this notification came from.

Maybe something like this?
```suggestion
        notify('RandomWallpaperGnome3: Wallpaper Download Failed!', errorMessage);
```

**Review by ifl0w — CHANGES_REQUESTED** [review]
Looks good, thanks! Just had the two small remarks.

File: src/notifications.ts (hunk 2 above)
Comment (Lucki) [reply src/notifications.ts]: Gave it a quick check, the title still fits in the line.

**Pull request merged**